lib/jar.ts

```diff
@@ -138,11 +141,13 @@ async function getMavenPackageInfo(sha1: string): Promise<MavenPackageInfo> {
             ),
           );
         }
-
         if (res.response.docs.length > 1) {
-          debug('Got multiple results for sha1, only returning first one');
+          const sha1Target = path.parse(targetPath).base;
```

Comment:
I'm not sure that this might be a case, but if targetPath doesn't contain pacakge name? (e.g. `/var/foo/bar/bar.jar`)? (@orsagie might know if this can be a case)

Reply:
I understand what you mean, it's a good point. Do you have any suggestion? Because I think that regardless of the solution this have to be solved, we cannot strong intake the 1st shai's response value as the right one :neutral_face:

Reply:
@dkontorovskyy is right, we can't count on the file name being the name of the package. it can be a default behaviour though. but let's have a fallback for this fallback (choosing the first element)

Reply:
Thank you for the feedback, I understand your point. I was trying to figure out a solution to avoid this fallback. So in this case to solve the test issue do you agree that will have to make it matches 
> org.netbeans.external:org-apache-commons-io@RELEASE113
even not being that we are really looking for?

Reply:
I am saying that we have two fallbacks :) first one will be to try and match the name of the file, and the second will be to return the first found occurrence (cause we want to return something)

Reply:
I see :)